.github/workflows/check-change-note.yml

```diff
@@ -8,6 +8,7 @@ on:
       - "*/ql/src/**/*.qll"
       - "!**/experimental/**"
       - "!ql/**"
+      - ".github/workflows/check-change-not.yml"
 
```

Comment:
Should change `*/ql/src/**/*.qll` to `*/ql/**/*.qll` (and same for `.ql`). Otherwise we're not running on changes that only affect the library packs.

Reply:
This was committed accidentally. I think we should exclude tests and example packs from here. So, I will add a line like this: `*/ql/lib/**/*.qll`. Same for `*.ql`. It's not really related to this change, but I'll add it anyway.